python/packages/aws-lambda-sdk/tests/fixtures/handlers.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import annotations
+
+
+def handler(*args, **kwargs) -> str:
+    return "Handler"
+
+
+def error(*args, **kwargs) -> str:
+    raise Exception("something went wrong")
+
+
+not_callable_handler = ""
```

Comment:
To have it compatible with the integration test engine we're going to use, Each test case should be configured with a different function file, and this looks to me as a lambda that exports multiple handlers that address many cases (?).

Also, in case of success, lambda should return an "ok" string (it's expected as default response in test engine), e.g.: https://github.com/serverless/console/blob/c1d85ac7185919f63bbb0d2de40864208e1e80bb/node/packages/aws-lambda-sdk/test/fixtures/lambdas/callback.js#L3

and, let's give function fixture files meaningful names (e.g. `dummy_module` or `handlers` do not express what use case they present)

Reply:
Great suggestion 👍 I've renamed the files according to their intended use and such that each of them serve a single purpose.